WeakAuras/Prototypes.lua

```diff
@@ -10702,6 +10705,163 @@ Private.event_prototypes = {
     end,
     automaticrequired = true
   },
+  ["Location"] = {
+    type = "unit",
+    events = {
+      ["events"] = {
+        "ZONE_CHANGED",
+        "ZONE_CHANGED_INDOORS",
+        "ZONE_CHANGED_NEW_AREA",
+        "VEHICLE_UPDATE",
+        "MINIMAP_UPDATE_ZOOM",
```

Comment:
Are you sure about MINIMAP_UPDATE_ZOOM and VEHICLE_UPDATE?

Reply:
I am sure about MINIMAP_UPDATE_ZOOM, because ZONE_CHANGED_INDOORS is not sufficient enough to check IsIndoors(). 
I am not sure about VEHICLE_UPDATE, I think I copied this from https://github.com/WeakAuras/WeakAuras2/blob/19bdbb7165fd2812c8e83f727f04c334c694357e/WeakAuras/Prototypes.lua#L1772
I don't know what the reason is for that being there, I have never found this event to be required for anything location related.

Reply:
Why isn't ZONE_CHANGED_INDOORS not enough? Can you explain?

Reply:
https://github.com/WeakAuras/WeakAuras2/assets/6916881/88970e86-0855-448e-9b48-aa20029d9b1a

https://github.com/WeakAuras/WeakAuras2/assets/6916881/e30c20f4-3775-4d43-9234-6daa7393e4ae

`MINIMAP_UPDATE_ZOOM` fires when you move from outdoor to indoor (and your minimap changes)
`ZONE_CHANGED_INDOORS` fires when you are already indoors and you change subzones.
Both events are required.